.env.dev

```diff
@@ -1,6 +1,6 @@
 PORT=5000
 NODE_ENV="development"
 JWT_SECRET="thisismysupersecrettokenjustkidding"
-DATABASE_URL="mongodb://localhost:27017/donut-development"
-SENDGRID_API_KEY = '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'
-SOCKET_PORT = 8810
\ No newline at end of file
+DATABASE_URL="mongodb://mongo:27017/donut-development"
```

Comment:
We need to switch between hostnames `mongo` and `localhost` for container and normal setup respectively.